src/CarbonAware.Aggregators/test/CarbonAwareAggregatorTests.cs

```diff
@@ -344,6 +343,7 @@ public async Task<double> CalculateAverageCarbonIntensityAsync_ValidTimeInterval
 
     [TestCase("2021-11-18T00:00:00Z", null, TestName = "AverageCI valid startTime, null endTime")]
     [TestCase(null, "2021-11-18T00:00:00Z", TestName = "AverageCI null startTime, valid endTime")]
+    [TestCase("2021-11-19T00:00:00Z", "2021-11-18T00:00:00Z", TestName = "AverageCI startTime greater than endTime")]
```

Comment:
```suggestion
    [TestCase("2021-11-19T00:00:00Z", "2021-11-18T00:00:00Z", TestName = "InvalidStartEndTimes: AverageCI startTime greater than endTime")]
```

Reply:
In fact, I would suggest renaming all the test cases so there is a clear distinction between test names. Adding at least some method tag would be good.

Reply:
> In fact, I would suggest renaming all the test cases so there is a clear distinction between test names. Adding at least some method tag would be good.

That's a bit ironic, since we'd also noticed a number of the existing "TestName" tags didn't reference the methods involved (or the outcome expected), just mainly the test inputs -- and admittedly we didn't set out to clean-up those.

Reply:
Yeah, I agree and faced a similar dilemma with our PR. However, we should probably demonstrate good stewardship by doing the renaming when we tinker with these files.

Reply:
after a chat with bderusha, will open a follow-up clean-up story